README.md

```diff
@@ -1,7 +1,9 @@
-# Form
-![crates.io badge](https://img.shields.io/crates/v/form.svg)
+[![crates.io](https://img.shields.io/crates/d/form.svg)](https://crates.io/crates/form)
+[![crates.io](https://img.shields.io/crates/v/form.svg)](https://crates.io/crates/form)
 [![CircleCI](https://circleci.com/gh/djmcgill/form/tree/main.svg?style=svg)](https://circleci.com/gh/djmcgill/form/tree/main)
```

Comment:
```
![example workflow](https://github.com/djmcgill/form/actions/workflows/.github/workflows/ci.yaml/badge.svg)
```
Should be it right instead of the circle badge? Bit hard to test without having run it first.

Reply:
Done. Copied from https://github.com/stm32-rs/stm32-rs